### ISSUE TITLE
New release PyPI installation failure

### ISSUE THREAD

**Issue by g-as** [mfogel/django-simple-email-confirmation#25]
@mdamien maybe you're already aware, but just in case...

New release is [here](https://pypi.python.org/pypi/django-simple-email-confirmation/) but un-installable via pip.

If version number is unspecified, 0.12 is installed

If 0.20 is specified :

``` shell
pip install django-simple-email-confirmation==0.20
Collecting django-simple-email-confirmation==0.20
  Could not find a version that satisfies the requirement django-simple-email-confirmation==0.20 (from versions: 0.9, 0.9.1, 0.10, 0.10.1, 0.10.2, 0.11, 0.12)
No matching distribution found for django-simple-email-confirmation==0.20
```

And BTW, thanks for stepping up man!

**Comment by mdamien** [comment]
wow, thanks for the tip, I need to fix that

**Comment by mdamien** [comment]
Fixed but I had to release a `0.21` version, so it's `pip install django-simple-email-confirmation==0.21`

**Comment by g-as** [comment]
a'ight. Thanks!
